recipes/openblas/all/conanfile.py

```diff
@@ -46,6 +46,8 @@ def _fortran_compiler(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if Version(self.version) >= "0.3.21":
```

Comment:
```suggestion
        # INFO: When no Fortran compiler is available, OpenBLAS builds LAPACK from an f2c-converted copy of LAPACK unless the NO_LAPACK option is specified
```

I would include a note, just to be clear about it.